### PR TITLE
`Development`: Add exam management modeling/programming/quiz exercise creation e2e tests

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] Tested on my local Artemis instance
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

The current exam management test only test the creation of a text exercises. Since those are not the only types of exercises that are available, the missing types should be added.  

### Description

This PR adds the missing exercise types modeling, programming and quiz to the exam management e2e tests. In order for the new tests to work, some minor element id changes where necessary. 

#### Code Review
- [x] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

**Comment by TheZoker** [comment]
Converting to draft, since E2E tests are failing and I'm not yet sure why...

**Comment by TheZoker** [comment]
There was a `true` value missing within the course creation for the exam. This lead to a permissions error within the programming exercises. This has been fixed and the tests are now running without any issue.

File: src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html [hunk 1]
```diff
@@ -59,7 +59,12 @@ <h5 class="font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
                             <fa-icon class="d-xl-none" [icon]="faCheckDouble"></fa-icon>
                             <span class="d-none d-xl-inline">{{ 'artemisApp.quizExercise.home.importLabel' | artemisTranslate }}</span>
                         </a>
-                        <a *ngIf="course?.isAtLeastEditor" [routerLink]="[exerciseGroup.id, 'quiz-exercises', 'new']" class="btn btn-info btn-sm me-1 mb-1" style="max-height: 44%">
```
Comment (JohannesStoehr) [inline src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html:62]: Did you remove the style="max-height: 44%" on purpose?

Comment (TheZoker) [reply src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html:62]: Ah I think that was accidentally removed during a rebase. Thanks for the hint, I re-added the style.

**Review by JohannesStoehr — DISMISSED** [review]
Code changes and new tests look good to me

**Comment by TheZoker** [comment]
Temp

**Review by JohannesStoehr — APPROVED** [review]
Rechecked after merges. Code still looks good

**Review by ge65cer — DISMISSED** [review]
LGTM

**Review by JohannesStoehr — DISMISSED** [review]
Additional exam tests look good to me

**Review by akesfeden — APPROVED** [review]
E2E tests pass on Bamboo and code LGTM.

**Review by akesfeden — APPROVED** [review]
Reapproving after author accidentally re-requested review.

**Review by JohannesStoehr — APPROVED** [review]
Merge commits look good to me

**Pull request merged**